src/test/models/users.model.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+import '../../server/models/users.model';
+
+/*
+ * Test for mongodb create.
+ */
+
+let should = require('should');
+let user = require('mongoose').model('user');
+
+describe('Users: models', function () {
+  describe('#create()', function () {
+    it('should create a new User', function (done) {
+      // Create a User object to pass to User.create()
+      var u = {
+        name: 'yoda'
+      };
+      user.create(u, function (err, createdUser) {
```

Comment:
Creating new database for test would be better for later or now. I'll add TODO in trello!